backend/test/institution_handler_test.py

```diff
@@ -581,6 +581,226 @@ def test_remove_admin_permission(self, verify_token, enqueue_task):
             first_user, second_inst.key.urlsafe(), permissions.DEFAULT_ADMIN_PERMISSIONS))
         self.assertTrue(has_permissions(
             first_user, fourth_inst.key.urlsafe(), permissions.DEFAULT_ADMIN_PERMISSIONS))
+    
+    @patch('handlers.institution_handler.enqueue_task')
+    @patch('utils.verify_token', return_value={'email': 'user@example.com'})
+    def test_delete_hierarchy_with_a_one_direction_link(self, verify_token, enqueue_task):
```

Comment:
To each test, you should add a comment to illustrate the configuration of this hierarchy. For example, look at RemoveAdminPermissionsInInstitutionHierarchyTest.py, line 53